docs/gettingStarted/quickStart.rst

```diff
@@ -381,12 +428,14 @@ on running Toil workflows on a cluster, see :ref:`runningAWS`.
 Running a CWL Workflow on AWS
 -----------------------------
 After having installed the ``aws`` and ``cwl`` extras for Toil during the :ref:`installation-ref` and set up AWS
-(see :ref:`prepare_aws-ref`),
-the user can run a CWL workflow with Toil on AWS.
+(see :ref:`prepare_aws-ref`), the user can run a CWL workflow with Toil on AWS.
+
+Also!  Remember to use the :ref:`destroyCluster` command when finished to destroy the cluster!  Otherwise things may not be cleaned up properly.
```

Comment:
Id change this to "Also, remember to... to destroy the cluster!"  I would do the same for the changes on line 468.

Reply:
The original has the link to the command so that the user specifically has a reference to what we're referring to.  There are many ways to destroy a cluster and we're trying to avoid having them just delete through the browser.